resources/js/processes/modeler/components/inspector/ErrorHandlingRetryAttempts.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div role="group">
+    <label for="retry_attempts">{{ $t('Retry Attempts') }}</label>
+    <b-form-input id="retry_attempts" type="number" v-model="config.retry_attempts"></b-form-input>
+    <small class="form-text text-muted">{{ helper }}</small>
+  </div>
+</template>
+
+<script>
+
+export default {
+  props: ['type'],
+  data() {
+    return {
+      config: {
+        retry_attempts: "",
+      },
+    }
+  },
+  watch: {
+    config: {
+      deep: true,
+      handler() {
+        this.setNodeConfig();
+      }
+    },
+  },
+  methods: {
+    node() {
+      return this.$root.$children[0].$refs.modeler.highlightedNode.definition;
+    },
+    getNodeConfig() {
+      const configString = _.get(this.node(), 'errorHandling', null);
+      if (configString) {
+        const config = JSON.parse(configString);
+        this.config.retry_attempts = _.get(config, 'retry_attempts', "");
+      }
+    },
+    setNodeConfig() {
+      const json = JSON.stringify({ retry_attempts: this.config.retry_attempts });
+      Vue.set(this.node(), 'errorHandling', json);
```

Comment:
This will overwrite the other settings for timeout and wait time. It needs to be merged, for example
```
const existingSetting = _.get(this.node(), 'errorHandling', {});
const json = JSON.stringify({ ...existingSetting, retry_attempts: this.config.retry_attempts });
```

Reply:
fixed